src/AbstractAlignFixerHelper.php

```diff
@@ -18,6 +18,8 @@
  * @author Carlos Cirello <carlos.cirello.nl@gmail.com>
  *
  * @internal
+ *
+ * @deprecated
  */
 abstract class AbstractAlignFixerHelper
```

Comment:
should we trigger a deprecation error for this class as well or is this enough together with the errors triggered by the two children?